### PR TITLE
Publish to Maven Central and make it work with Spark 3.0.1

### PR DESCRIPTION
**Pull request by arielshaqed**
Pushing to keep working; might not _entirely_ work... :-/

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1 +1,2 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
+addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
```
Comment (johnnyaug) [inline project/plugins.sbt]: Do we want this?

File: build.sbt [hunk 2]
```diff
@@ -24,11 +24,11 @@ core / libraryDependencies ++= Seq("org.rocksdb" % "rocksdbjni" % "6.6.4",
   "org.apache.spark" %% "spark-sql" % "3.0.1" % "provided",
   "com.thesamet.scalapb" %% "sparksql-scalapb" % "0.10.4" % "protobuf",
   "com.thesamet.scalapb" %% "scalapb-runtime" % "0.10.4" % "protobuf",
-  "org.apache.hadoop" % "hadoop-aws" % "2.10.1",
-  "org.apache.hadoop" % "hadoop-common" % "2.10.1",
+  "org.apache.hadoop" % "hadoop-aws" % "2.7.2",
+  "org.apache.hadoop" % "hadoop-common" % "2.7.2",
```
Comment (johnnyaug) [inline build.sbt]: 2.7.2 is very old. I'm working with 2.7.7 and it works (for now)

File: project/plugins.sbt (hunk 1 above)
Comment (arielshaqed) [reply project/plugins.sbt]: [The fine manual says we do](https://github.com/sbt/sbt-pgp#setup)!  I imagine the reasoning is something like this...
> _I know I can sign and publish with sbt-pgp version 2.1.2.  I might not be able to sign and publish with some other version of sbt-pgp.  So let's drag it in._
Given that we do the same with sbt-assembly, which also says to include it, I think this makes sense.

**Review by johnnyaug — APPROVED** [review]

**Comment by arielshaqed** [comment]
Thanks!

**Pull request merged**